spec/models/king_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe King, type: :model do
+  describe 'valid_move?' do
+    it 'should return true if moving one square up' do
+      FactoryGirl.create(:game)
+      king = King.create(x_position: 6, y_position: 3)
+      expect(king.valid_move?(6, 4)).to eq true
+    end
+
+    it 'should return true if moving one square left' do
+      FactoryGirl.create(:game)
+      king = King.create(x_position: 6, y_position: 3)
+      expect(king.valid_move?(5, 3)).to eq true
+    end
+
+    it 'should return true if moving one square diagonal' do
+      FactoryGirl.create(:game)
+      king = King.create(x_position: 6, y_position: 3)
+      expect(king.valid_move?(7, 2)).to eq true
+    end
+
+    it 'should return false if moving more than one square up' do
+      FactoryGirl.create(:game)
```

Comment:
can remove this line in each test (create game).

Use factory girl to create the `king` and have it automatically belong to a game.

`king = FactoryGirl.create(:king, x_position...)`